### PR TITLE
Update release to be compatible with new build scripts

### PR DESCRIPTION
**Pull request by johnoliver**
1. Change datestamp on file to "YYYY-MM-DD-hh-mm"
1. Pin timestamp of all released files to be consistent and set to UTC time

### CODE REVIEW

File: sbin/Release.sh [hunk 1]
```diff
@@ -14,44 +14,48 @@
...
+    mv "${file}.sha256.txt" "${newName}.sha256.txt"
+
+    # Fix checksum file name
+    sed -i -r "s/^([0-9a-fA-F ]+).*/\1${newName}/g" "${newName}.sha256.txt"
+
+    FILE_VERSION=${BASH_REMATCH[1]};
+    FILE_ARCH=${BASH_REMATCH[2]};
+    FILE_OS=${BASH_REMATCH[3]};
+    FILE_VARIANT=${BASH_REMATCH[4]};
+    FILE_EXTENSION=${BASH_REMATCH[6]};
```
Comment (karianna) [inline sbin/Release.sh]: what happened to BASH_REMATCH[5]?

Comment (johnoliver) [reply sbin/Release.sh]: It is the timestamp of the file, as per the server that built the file, so not consistent across all the builds, so we dont care so much what timestamp the build server put on it

File: sbin/Release.sh [hunk 2]
```diff
@@ -14,44 +14,48 @@
 #
 
 npm install
-# Expect a naming convention for build artifacts to follow this pattern:
-# OpenJDK<version>_<arch>_<os>_<timestampOrTag>.<extension>
-# Examples: OpenJDK8_x64_Windows_201813060547.zip, 
-#           OpenJDK8_x64_LinuxLH_201813060547.tar.gz, 
-#           OpenJDK10_aarch64_Linux_201813060547.tar.gz.sha256.txt,
-#           OpenJDKamber_x64_Linux_201813061304.tar.gz	
+
```
Comment (sxa555) [inline sbin/Release.sh]: Can you put an example back in to show what we're translating i.e. from something YYYYY-DD-MM to YYYY-MM-DD - not obvious to the casual user what's going on here (and you've removed the examples that were there previously)

File: sbin/Release.sh [hunk 3]
```diff
@@ -14,44 +14,48 @@
...
+  if [[ $file =~ $regexArchivesOnly ]];
   then
-    mv $file "Open${VERSION}_${ARCH}_${OS}_${TS_TAG}${SHA_EXT}"
-  else
-    mv $file "Open${VERSION}_${ARCH}_${OS}_${TS_TAG}.${EXTENSION}"
+    newName=$(echo "${file}" | sed -r "s/${timestampRegex}/$TIMESTAMP/")
+
+    # Rename archive and checksum file with now timestamp
+    mv "${file}" "${newName}"
```
Comment (sxa555) [inline sbin/Release.sh]: I think a `echo Renaming ${file} to ${newName}` would be good in here, just to make it clear what's going on if anyone's trying to debug

**Pull request merged**